smartsim/ml/data.py

```diff
@@ -471,12 +482,8 @@ def _update_samples_and_targets(self) -> None:
 
             # Strictly greater, because next_index is 0-based
             if list_length > self.next_indices[uploader_idx]:
-                indices = [
-                    idx
-                    for idx in range(
-                        self.next_indices[uploader_idx], list_length, self.num_replicas
-                    )
-                ]
+                start, end = self.next_indices[uploader_idx], list_length
```

Comment:
please don't do two assignments on the same line

Reply:
zero reason to have `end` when i had `list_length`. removed!